.travis.yml

```diff
@@ -8,26 +8,32 @@ php:
   - 7.0
   - hhvm
 
-matrix:
-  allow_failures:
-    - php: 7.0
-    - php: hhvm
+sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - vendor
```

Comment:
I'll finally remove vendor cache because of this issue: https://github.com/composer/composer/issues/4147#issuecomment-115258930